response/template_response.json

```diff
@@ -0,0 +1,191 @@
+            "managementIpAddress": null,
+            "configFile": null,
+            "serialNumber": null,
+            "asmGUID": null,
+            "relatedComponents": {
+                "7f54bbd1-6c5d-43d8-8241-19226e10f519": "PowerFlex Cluster"
+            },
+            "resources": [],
+            "refId": null,
+            "cloned": false,
+            "clonedFromId": null,
+            "manageFirmware": false,
+            "brownfield": false,
+            "instances": 3,
+            "clonedFromAsmGuid": null,
+            "ip": null
+        }
+    ],
+    "category": "Software Only",
+    "allUsersAllowed": false,
+    "assignedUsers": [],
+    "manageFirmware": true,
+    "useDefaultCatalog": false,
+    "firmwareRepository": {
+        "id": "8aaaee208c8c467e018cd37813250614",
+        "name": "PowerFlex 4.5.1.0",
+        "sourceLocation": null,
+        "sourceType": null,
+        "diskLocation": null,
+        "filename": null,
+        "md5Hash": null,
+        "username": null,
+        "password": null,
+        "downloadStatus": null,
+        "createdDate": null,
+        "createdBy": null,
+        "updatedDate": null,
+        "updatedBy": null,
+        "defaultCatalog": false,
+        "embedded": false,
+        "state": null,
+        "softwareComponents": [],
+        "softwareBundles": [],
+        "deployments": [],
+        "bundleCount": 0,
+        "componentCount": 0,
+        "userBundleCount": 0,
+        "minimal": false,
+        "downloadProgress": 0,
+        "extractProgress": 0,
+        "fileSizeInGigabytes": null,
+        "signedKeySourceLocation": null,
+        "signature": null,
+        "custom": false,
+        "needsAttention": false,
+        "jobId": null,
+        "rcmapproved": false
+    },
+    "licenseRepository": null,
+    "configuration": null,
+    "serverCount": 3,
+    "storageCount": 0,
+    "clusterCount": 1,
+    "serviceCount": 0,
+    "switchCount": 0,
+    "vmCount": 0,
+    "sdnasCount": 0,
+    "brownfieldTemplateType": "NONE",
+    "networks": [
+        {
+            "id": "8aaaee038c939c14018cd3defc590004",
+            "name": "LGLOU",
+            "description": "",
+            "type": "SCALEIO_DATA",
+            "vlanId": 1,
+            "static": true,
+            "staticNetworkConfiguration": {
+                "gateway": "10.1039.1",
+                "subnet": "255.255.255.0",
+                "primaryDns": "10.10.13.8",
+                "secondaryDns": null,
+                "dnsSuffix": null,
+                "ipRange": [
+                    {
+                        "id": "8aaaee328cfd27d1018d5953e3380a0d",
+                        "startingIp": "10.1039.122",
+                        "endingIp": "10.1039.132",
+                        "role": null
+                    }
+                ],
+                "ipAddress": null,
+                "staticRoute": null
+            },
+            "destinationIpAddress": "10.1039.0"
```

Comment:
Fairly small detail, but can we make sure all the numbers in the IPs are <= 255? I see several places where the address has a number >255, which is not valid.

Reply:
Updated.